Sources/Bookmarks/BookmarksModel.xcdatamodeld/.xccurrentversion

```diff
@@ -3,6 +3,6 @@
 <plist version="1.0">
 <dict>
 	<key>_XCCurrentVersionName</key>
-	<string>BookmarksModel 4.xcdatamodel</string>
+	<string>BookmarksModel 5.xcdatamodel</string>
```

Comment:
Note that I'd expect this to trigger (again) the fix for migrating the favorites on the platforms - I don't think it's harmful, just something to remember.